core/trino-main/src/test/java/io/trino/execution/scheduler/TestHashDistributionSplitAssigner.java

```diff
@@ -314,6 +321,150 @@ PARTITIONED_2, new OutputDataSizeEstimate(ImmutableLongArray.of(1, 1, 1))))
                 .run();
     }
 
+    @Test
```

Comment:
a test with missin `OutputDataSizeEstimate`? Or is there and I missed it?

Reply:
The missing estimates are covered by `testMissingEstimates`. Or do you think we need a test specific to splitting?